src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/extension/env/EnvExtension.groovy

```diff
@@ -186,6 +215,100 @@ class EnvExtension extends AbstractGlobalExtension implements SpringContextListe
                     server42IslRtt = (sw.prop.server42IslRtt == null ? "AUTO" : (sw.prop.server42IslRtt ? "ENABLED" : "DISABLED"))
                 })
             }
+
+            log.info("Starting topology-related HC: lab_id=" + topo.labId)
```

Comment:
Why do you copy-paste this healthcheck instead of reusing it (if it is needed at all)?

Reply:
In this case we check topology readiness per topology from topologies pool after its creation. In the case of rebuilding topology, only this part will be checked and the HC above #110 will be skipped.

Reply:
Can we move this to method with List<Closure> parameter to not repeat code?

Reply:
done